src/applications/vaos/tests/containers/TypeOfCarePage.unit.spec.jsx

```diff
@@ -182,11 +181,11 @@ describe('VAOS <TypeOfCarePage>', () => {
   it('should NOT display alert message once user clicks the update address button using redux state', () => {
     const store = createTestStore({});
 
-    const router = {
+    const history = {
       push: sinon.spy(),
     };
-    const { getByText, queryByText } = renderInReduxProvider(
-      <TypeOfCarePage router={router} />,
+    const { getByText, queryByText } = renderWithStoreAndRouter(
+      <TypeOfCarePage history={history} />,
```

Comment:
I think you had mentioned it when we discussed this PR, but this new `renderWithStoreAndRouter()` helper would handle the creation and passing down the `history` prop right, so we wouldn't need the declaration on line 184, right?

Reply:
Correct, yeah, I think I just missed this one